packages/webservice/app/src/main/resources/db/changelog/changelog.xml

```diff
@@ -108,4 +108,27 @@
         );
         </sql>
     </changeSet>
+    <changeSet author="admin" id="2">
+        <sql>
+          CREATE TABLE TableExport (
+             id varchar(36) not null,
+             createdOn timestamp,
+             principalName varchar(255),
```

Comment:
How does this relate to the user table?

Reply:
the principal name is the user principal name (should ideally match the user table).

Reply:
Could we add a foreign key to the user table?

Reply:
Is there an entry in the `User` table available for No-Authentication scenario? When we use it for a demo, `principalName` is going to be null.

Reply:
yes, we add "admin" in the demo case

Reply:
Verified that the `principalName` is not null for AsyncQuery in Yavin App, its set to `admin` in case of No Auth scenario. Adding a Foreign Key for AsyncQuery and TableExport should be fine.

Reply:
https://github.com/yahoo/yavin/blob/5e5aa18cabb5bd3ee63a8cbf0436d2a3ecf96be7/packages/webservice/app/src/main/kotlin/com/yahoo/navi/ws/filters/AuthFilter.kt#L24